src/v2/Apps/Order/Components/AddressModal.tsx

```diff
@@ -222,16 +229,21 @@ export const AddressModal: React.FC<Props> = ({
             if (address?.internalID) {
               onDeleteAddress(address.internalID)
             }
+            handleClearAddress()
           },
           text: "Delete",
         }}
         secondaryCta={{
           action: () => {
             setShowDialog(false)
+            handleClearAddress()
           },
           text: "Cancel",
         }}
-        onClose={() => setShowDialog(false)}
+        onClose={() => {
+          setShowDialog(false)
+          handleClearAddress()
```

Comment:
I see `handleClearAddress` only does execute `onClearAddress`... could we use `onClearAddress()` directly here?

Reply:
There's a pattern in force that suggests "onAction" a passed prop and "handleAction" is used within the component. I'm just following that paradigm for the sake of consistency.